hackerrank/1-week-prep/day-6/simple-text-editor.py

```diff
@@ -0,0 +1,58 @@
+class stack:    
+    def __init__(self):
+        self._values = []
+        self._size = 0
+        
+    def is_empty(self):
+        return self._size == 0
+        
+    def push(self, value):
+        self._size += 1
+        self._values.append(value)
+    
+    def top(self):
+        return self._values[self._size-1]
+    
+    def pop(self):
+        self._size -= 1
+        self._values.pop()
+
+class text_editor:
+    def __init__(self):
+        self._content = ''
+        self._content_stack = stack()
+        self._content_stack.push(self._content)
```

Comment:
Great 💡

Reply:
Very clever to just add the current state of the string to the stack, I wish I had thought of that.